management-account/terraform/sso-scim.tf

```diff
@@ -8,3 +8,11 @@ module "scim" {
   sso_identity_store_id = local.sso_admin_identity_store_id
   not_dry_run           = true
 }
+
+module "entraid_scim" {
+  # tflint-ignore: terraform_module_pinned_source
```

Comment:
Ideal for it to be pinned - but I'm happy to keep it like this while it's still under development 🚀

Reply:
Yeah it would be awfully convenient for now if that's ok.